data/recipes/gcp_forensics.json

```diff
@@ -11,22 +11,25 @@
           "remote_instance_name": "@instance",
           "incident_id": "@incident_id",
           "zone": "@zone",
+          "create_analysis_vm": "@analysis_vm",
           "disk_names": "@disks",
           "all_disks": "@all_disks",
           "boot_disk_size": "@boot_disk_size",
           "boot_disk_type": "@boot_disk_type",
-          "cpu_cores": 16,
+          "cpu_cores": "@cpu_cores",
           "image_project": "ubuntu-os-cloud",
           "image_family": "ubuntu-1804-lts"
       }
   }],
   "args": [
       ["remote_project_name", "Name of the project containing the instance / disks to copy ", null],
-      ["incident_id", "Incident ID to label the VM with.", null],
+      ["analysis_project_name", "Name of the project where the analysis VM will be created", null],
```

Comment:
I think like you may have wanted to harmonize this and the `gcp_turbinia_disk_copy_ts.json` recipe. It would maybe be better to revisit this in a future PR than to squeeze it in here

Reply:
Got it. I will make the analysis_project_name optional again.

Reply:
Oh also I remember something else,  gcloud doesn't check if analysis_project_name exist or not, if not it will default to none and then error when it try to print the analysis vm name. I was buggy, I will ad additional condition to make it work as aoptional.